Current jobs tracking, run execs, EOF

marwtki opened the pull request.

marwtki commented:
@disaacson  I did not commit the changes to build.sbt or Build.scala

disaacson commented:
What were the changes to build.sbt? Are those necessary or is this pull request good to go?

marwtki commented:
In build.sbt we added 
```connectInput in run := true```
and updated the version to 1.14-SNAPSHOT. Does the version update need to be part of the pull request?
